extensions/amp-izlesene/0.1/validator-amp-izlesene.protoascii

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-izlesene
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "SCRIPT"
+  spec_name: "amp-izlesene extension .js script"
+  mandatory_parent: "HEAD"
+  unique: true
```

Comment:
Actually, we're moving this to using `satisfies` and `requires` instead of `also_requires_tag`. So instead of putting `also_requires_tag: amp-izlesene` please put above under `spec_name`, `satisfies: "amp-izlesene extension .js script"` and below in `tag_name: "AMP-IZLESENE"` replace the `also_requires_tag` with `requires: "amp-izlesene extension .js script"`. See this [PR](https://github.com/ampproject/amphtml/pull/7493/files#diff-424eda0aa64626104e3c79ac66900fa4) as an example.